amp-pwa-reader/gulpfile.js

```diff
@@ -117,16 +127,16 @@ function watch() {
     ui: false
   });
 
-  gulp.watch(paths.scripts.src, gulp.series(scripts, inline));
-  gulp.watch(paths.styles.src, gulp.series(styles, inline, injectManifest));
+  gulp.watch(paths.scripts.src, gulp.series(jsScripts, inline));
```

Comment:
let's choose some other names for jsScripts and scripts. Not obvious when reading here what the diff is. a script is (almost) always JS on the web :)

Reply:
definitely better.